### PR TITLE
update to gradle 5.4.1

### PR DESCRIPTION
**Pull request by cdietrich**
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: gradlew [hunk 1]
```diff
@@ -1,5 +1,21 @@
 #!/usr/bin/env sh
 
+#
+# Copyright 2015 the original author or authors.
```
Comment (szarnekow) [inline gradlew]: Do we need to update the gradlew in all the other projects, too?

Comment (szarnekow) [reply gradlew]: Or is that because of 5.3.1 -> 5.4.1 vs 5.2.1 -> 5.4.1

Comment (cdietrich) [reply gradlew]: there is only one gradle project

Comment (cdietrich) [reply gradlew]: yes there other versions where not updated when in xtext
thus here the scripts are changed too

**Pull request merged**